### PR TITLE
adding support for ltm policy rule http_method condition type

### PR DESCRIPTION
**Pull request by RavinderReddyF5**

### CODE REVIEW

File: ansible_collections/f5networks/f5_modules/plugins/modules/bigip_policy_rule.py [hunk 1]
```diff
@@ -449,9 +458,12 @@
         version_added: "1.10.0"
       event:
         description:
-          - Events on which conditions such as SSL Extension can be triggered.
+          - Events on which conditions type match rules can be triggered.
+          - Supported only for c(http_header),c(http_method),c(ssl_extension),c(tcp).
```
Comment (wojtek0806) [inline ansible_collections/f5networks/f5_modules/plugins/modules/bigip_policy_rule.py:462]: incorrect doc formatting, use C() also spaces between each name

File: ansible_collections/f5networks/f5_modules/plugins/modules/bigip_policy_rule.py [hunk 2]
```diff
@@ -420,6 +422,13 @@
           - This parameter is only valid with the C(http_header) type.
         type: str
         version_added: "1.8.0"
+      method_matches_with_any:
+        description:
+          - A list of strings of characters the HTTP Method value should match.
+          - This parameter is only valid with the C(http_method) type.
+        type: list
+        elements: str
+        version_added: "1.9.1"
```
Comment (wojtek0806) [inline ansible_collections/f5networks/f5_modules/plugins/modules/bigip_policy_rule.py:431]: its not version 1.9.1 it will be version 1.10.0

**Review by wojtek0806 — CHANGES_REQUESTED** [review]

File: ansible_collections/f5networks/f5_modules/plugins/modules/bigip_policy_rule.py (hunk 2 above)
Comment (RavinderReddyF5) [reply ansible_collections/f5networks/f5_modules/plugins/modules/bigip_policy_rule.py]: changed

File: ansible_collections/f5networks/f5_modules/plugins/modules/bigip_policy_rule.py (hunk 1 above)
Comment (RavinderReddyF5) [reply ansible_collections/f5networks/f5_modules/plugins/modules/bigip_policy_rule.py]: updated changes

**Pull request merged**